modin/core/execution/ray/common/utils.py

```diff
@@ -217,7 +217,17 @@ def initialize_ray(
         _move_stdlib_ahead_of_site_packages
     )
     ray.worker.global_worker.run_function_on_all_workers(_import_pandas)
-    num_cpus = int(ray.cluster_resources()["CPU"])
+    num_cpus = ray.cluster_resources().get("CPU", None)
```

Comment:
```suggestion
    num_cpus = ray.cluster_resources().get("CPU", None) or 4
```

Reply:
The reason I did it like this is because I wanted to warn users that we were unable to determine how many `CPU`s there were and were relying on an internal heuristic - should we keep it this way, or change it to the more streamlined version you propose?